lib/ansible/runner.rb

```diff
@@ -217,14 +230,19 @@ def async?(ansible_runner_method)
         ansible_runner_method == "start"
       end
 
-      def runner_params(base_dir, ansible_runner_method, playbook_or_role_args)
+      def runner_params(base_dir, ansible_runner_method, playbook_or_role_args, verbosity)
         runner_args = playbook_or_role_args.dup
 
         runner_args.delete(:roles_path) if runner_args[:roles_path].nil?
 
         runner_args[:role_skip_facts] = nil if runner_args.delete(:role_skip_facts)
         runner_args[:ident] = "result"
 
+        if verbosity.to_i > 0
```

Comment:
The .to_i is being done here presumably to deal with nil or other, but the next line doesn't have it and would probably blow up.